TodoList/Extensions/UIViewControllerExtensions.swift

```diff
@@ -2,21 +2,41 @@ import UIKit
 
 extension UIViewController {
     
-    /// Adds a subview to the calling `UIViewControler` and constrains it to the edges
+    /// Adds the specified subview to the calling `UIViewControler` and constrains it to the edges.
     ///
     /// - Parameter subview: view to add as subview and constraint.
     /// - Parameter margin: Optional margin inset to all of the edges of the subview. Defaults to 0.
+    ///
     func addSubViewWithFillConstraints(_ subview: UIView, margin: CGFloat = 0) {
         view.addSubViewWithFillConstraints(subview, margin: margin)
     }
     
+    /// Adds the specified view controller as a child of the current view controller.
+    ///
+    /// This convenience function runs the following code:
+    ///
+    ///     child.translatesAutoresizingMaskIntoConstraints = false
+    ///     addChildViewController(child)
+    ///     view.addSubview(child)
+    ///     child.didMove(toParentViewController: self)
```

Comment:
This is not a good idea. If this is public you can just see the code, if it isn't you're exposing the implementation details. Either way, don't do this.

You can either not mention what it does at all or explain what it does with words but don't show the code in the comments.